docs/docs/recipes/deploying-your-site.md

```diff
@@ -1,132 +1,132 @@
 
-### Directions
+### Wskazówki
 
-1. Login into Now CLI using `now login`
+1. Zaloguj się do Now CLI wykorzystując `now login`
 
-2. Change to the directory of your Gatsby.js application in the Terminal if you aren't already there
+2. Zmień katalog swojej aplikacji Gatsby w Terminalu jeśli jeszcze tam nie widnieje.
 
-3. Run `now` to deploy it
+3. Uruchom `now` aby ją wdrożyć
 
-### Additional resources
+### Dodatkowe źródła
 
-- [Deploying to ZEIT Now](/docs/deploying-to-zeit-now/)
+- [Wdrożenie na ZEIT Now](/docs/deploying-to-zeit-now/)
 
-## Deploying to Cloudflare Workers
+## Wdrożenie na platformie Cloudflare Workers
 
-Use [`wrangler`](https://developers.cloudflare.com/workers/tooling/wrangler/) to deploy your Gatsby application globally without leaving the command-line interface.
+Wykorzystaj [`wrangler`](https://developers.cloudflare.com/workers/tooling/wrangler/) aby wdrożyć swoją aplikację Gatsby bez opuszczania interfejsu konsoli.
 
-### Prerequisites
+### Wymagania
 
-- An account on [Cloudflare](https://dash.cloudflare.com/sign-up)
-- A [Workers Unlimited plan](https://developers.cloudflare.com/workers/about/pricing/) for \$5/month to enable the KV store, which is required to serve the Gatsby files.
-- A [Gatsby site](/docs/quick-start) set up with Gatsby's CLI
-- [wrangler](https://developers.cloudflare.com/workers/tooling/wrangler/install/) installed globally (`npm i -g @cloudflare/wrangler`)
+- Konto [Cloudflare](https://dash.cloudflare.com/sign-up)
+- [Plan Workers Unlimited](https://developers.cloudflare.com/workers/about/pricing/) za \$5/miesięcznie aby włączyć KV, które jest wymagane aby serwować pliki Gatsby.
+- [Strona Gatsby](/docs/quick-start) ze skonfigurowanym Gatsby CLI
+- Zainstalowany globalnie(`npm i -g @cloudflare/wrangler`) [wrangler](https://developers.cloudflare.com/workers/tooling/wrangler/install/)
 
-### Directions
+### Wskazówki
 
-1. Build your Gatsby application using `gatsby build`
-2. Run `wrangler config` where you'll be prompted for your [Cloudflare API token](https://developers.cloudflare.com/workers/quickstart/#api-token)
-3. Run `wrangler init --site`
-4. Configure `wrangler.toml`. First add [account ID](https://developers.cloudflare.com/workers/quickstart/#account-id-and-zone-id) field and then either
-   1. A free workers.dev domain by setting `workers_dev = true`
-   2. A custom domain on Cloudflare by setting `workers_dev = false`, `zone_id = "abdc..` and `route = customdomain.com/*`
-5. In `wrangler.toml` set `bucket = "./public"`
-6. Run `wrangler publish` and your site will be deployed in seconds!
+1. Zbuduj swoją aplikację Gatsby używając `gatsby build`
+2. Uruchom `wrangler config`, zostaniesz poproszony o swój [Cloudflare API token](https://developers.cloudflare.com/workers/quickstart/#api-token)
+3. Uruchom `wrangler init --site`
+4. Skonfigutuj `wrangler.toml`. Na początku dodaj pole [account ID](https://developers.cloudflare.com/workers/quickstart/#account-id-and-zone-id) a następnie
```

Comment:
Skonfigutuj `wrangler.toml`.
change to
Skonfiguruj `wrangler.toml`.
// small typo